src/ZoneServer/World/Items/Item.cs

```diff
@@ -97,6 +97,21 @@ public int Amount
 		/// </summary>
 		public Properties Properties { get; } = new Properties("Item");
 
+		/// <summary>
+		/// Gets or sets an expiration date on the item
+		/// </summary>
+		public DateTime ExpirationDate { get; private set; } = DateTime.MaxValue;
```

Comment:
This property doesn't appear to get used yet. Did you include it for future use and because of its use in `IsExpired`, which will never return true in the current code?

Reply:
So this was one of those partially cut out of the "card synthesis" code because I'm not sure if we want to go full property on the implementation or not. You can't use an expired material item to upgrade the card. I can remove the expiring item check till we decide what we want to do.

`
{ namespace: "Item", name: "LifeTime", type: "Number", id: 12614 },
{ namespace: "Item", name: "ItemLifeTimeOver", type: "Number", id: 31586 },
{ namespace: "Item", name: "ItemLifeTime", type: "String", id: 31587 },
`
LifeTime defines the duration (in seconds) (or if an item can expire)
ItemLifeTime is the "calculated" date of expiration
ItemLifeTimeOver is a (0 or 1) if the item is expired.